### DISCUSSION TITLE
Woodpecker daily cron builds get stuck

### DISCUSSION THREAD

**Discussion by martint17r** [woodpecker-ci/woodpecker#3373] [Ask a Question]
### Component

agent

### Describe the bug

After about two days our @daily cron builds get stuck and error immediately or timeout after 60 minutes:
- fatal: could not read Username for 'https://git...de': No such device or address\nexit status 128
- Error response from daemon: unable to find network with name or ID wp_01hpbtc4ccn66zzwk22mvd33a0_0_default: network not found

Restarting the agent resolves the problem and the @daily cron jobs work for a few days.

What actions would you recommend me to take to debug this problem?

### System Info

```shell
{"source":"https://github.com/woodpecker-ci/woodpecker","version":"2.3.0"} - also occured with 2.2.0
Debian bookworm
podman 4.3.1
podman server and agent processes are managed through systemd units created by run podman generate.
gitea 1.21.5
WOODPECKER_AUTHENTICATE_PUBLIC_REPOS=true
MariaDB 10.11.6
```

### Validations

- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]

**Comment by martint17r** [comment]
This problem has not occured after we upgraded to 2.4.1

**Comment by martint17r** [comment]
The problem reoccured with 2.6.0 - now the error is "pipeline definition not found".

**Comment by martint17r** [comment]
With 2.8.0 it also occurs - if we distribute the builds manually the problems occur less often.

**Comment by zc-devs** [comment]
That is not the debug is needed here, but correct implementation of Podman backend.
#4366

**Comment by qwerty287** [comment]
As there's no official podman support (yet), I'll close this
